opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1144,6 +1148,28 @@ def error_status_test(context):
             .start_as_current_span("root")
         )
 
+    def test_record_exception_fqn(self):
+        span = trace._Span("name", mock.Mock(spec=trace_api.SpanContext))
+        module_name = "dummy.module"
+        with mock.patch.object(DummyError, "__module__", module_name):
```

Comment:
What do you get if you don't mock it? If it's stable there's no need to mock

Reply:
Just to have a custom pathing in module name instead of `tests.trace.test_trace.DummyError` to assert.

Reply:
I can also use `self.__module__` to get the module name to assert and remove the mock